docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -177,6 +177,59 @@ a few caveats:
   - Enable using Apache Arrow serialization when reading data from BigQuery.
     Please read this [section](bigquery-arrow-serialization-support) before using this feature.
   - `true`
+* - `bigquery.channel-pool.initial-size`
+  - The initial size of the connection pool, also known as a channel pool,
+    used for gRPC communication.
+  - `1`
+* - `bigquery.channel-pool.min-size`
+  - The absolute minimum size of the connection pool, also known as a channel
```

Comment:
```suggestion
  - The minimum size of the connection pool, also known as a channel
```

Reply:
Also ... size in what .. maybe better 

The minimum number of connections in the connection pool, ...